packages/ember-htmlbars/lib/keywords/closure-component.js

```diff
@@ -74,12 +74,14 @@ export function isComponentCell(component) {
 }
 
 function createNestedClosureComponentCell(componentCell, params, hash) {
+  let newHash = assign(Object.create(null), hash);
```

Comment:
Currently, we do not use inheritance in `hash` objects (AFAIK). Do we really need to introduce that here?

Also, if we do, it should use `new EmptyObject()` instead (`import EmptyObject from 'ember-metal/empty_object'`).